src/pages/lib-v1/storage/configureaccess/q/platform/[platform].mdx

```diff
@@ -26,3 +26,102 @@ import android2 from '/src/fragments/lib-v1/storage/native_common/configureacces
 import flutter3 from '/src/fragments/lib-v1/storage/native_common/configureaccess/common.mdx';
 
 <Fragments fragments={{ flutter: flutter3 }} />
+
+<InlineFilter filters={['js']}>
+
+Storage module can manage files with three different access levels; `guest`, `protected` and `private`. The Amplify CLI configures three different access levels on the storage bucket: guest, protected and private. When you run `amplify add storage`, the CLI will configure appropriate IAM policies on the bucket using a Cognito identity pool Role. You will have the option of adding CRUD (Create/Update, Read and Delete) based permissions as well, so that Authenticated and Guest users will be granted limited permissions within these levels.
+
+If you had previously enabled user sign-in by running `amplify add auth` in your project, the policies will be connected to an `Authenticated Role` of the identity pool which has scoped permission to the objects in the bucket for each user identity. If you haven't configured user sign-in, then an `Unauthenticated Role` will be assigned for each unique user/device combination, which still has scoped permissions to just their objects.
+
+* Guest: Accessible by all users of your app. Files are stored under the `public/` path in your S3 bucket.
```

Comment:
Is this correct? "Guest" access is stored under the "public/" path? Shouldn't they be consistent?

Reply:
@renebrandel This is correct. Other libraries(Flutter, Swift) also use `Guest` access and stores file under `public/` path. The naming of `guest` preferred becaue it reduces the confusion it might caused from `public`. However, we cannot change the file path in S3 from `public/` as it would be a breaking change. 

I don't think other libraries doc, although also use `guest`, do not mention about the file path being `public/`, I will remove this sentence.

Reply:
The `guest` access level actually stores files under the prefix `public/` in the bucket. Native platform libraries adapted this odd mapping, I think, for backwards compatibility with the buckets created with using Amplify JS back then when it used access level `public`.

Amplify JS has now made the change to align with other platform libraries.